README.md

```diff
@@ -69,11 +69,6 @@ Bundle:
 $ yarn webpack
 ```
 
-Start Redis:
```

Comment:
You could probably leave this in if they want to do local testing.
maybe add a sentence to say if you would like to use a local version then you can follow this guide for running a local redis or you can connect to the remote redis instance

Reply:
added command back to readme, resolving